.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 defaults: &defaults
   working_directory: ~/repo
   docker:
-    - image: circleci/node:9.7.1
+    - image: circleci/node:latest
```

Comment:
I bumped us up to `circleci/node:latest` because `9.7.1` is _OLD_ and only included `git@2.1.4`... which doesn't include the `git worktree` command I wanted to use.